pkgs/jni/lib/src/jni.dart

```diff
@@ -35,7 +35,7 @@ DynamicLibrary _loadDartJniLibrary({String? dir, String baseName = "dartjni"}) {
   try {
     final dylib = DynamicLibrary.open(libPath);
     return dylib;
-  } on Error {
+  } catch (_) {
```

Comment:
Wait, isn't this even worse than catching `Error`?

Reply:
Solves the lint though 🤷  we're basically replacing the error thrown by `DynamicLibrary.open` with a custom `HelperNotFound` error so it's easier to resolve the problem.